vendor/vendor.json

```diff
@@ -1873,11 +1915,36 @@
 			"versionExact": "v3.1.1"
 		},
 		{
-			"checksumSHA1": "6VyMn387vNTCIxqwLuScKb+CDgQ=",
-			"origin": "github.com/elastic/beats/vendor/github.com/gogo/protobuf/proto",
+			"checksumSHA1": "A5e5CHQlZHu29CraUbhEAuxtrf0=",
+			"path": "github.com/gogo/googleapis/google/api",
+			"revision": "b8d18e97a9a193c846d32143391f6033a9d69849",
+			"revisionTime": "2019-09-05T23:40:13Z"
+		},
+		{
+			"checksumSHA1": "NeKpDVhUY91w23M3j6STWeGlvQ8=",
+			"path": "github.com/gogo/protobuf/gogoproto",
+			"revision": "5628607bb4c51c3157aacc3a50f0ab707582b805",
+			"revisionTime": "2019-10-14T06:15:17Z"
+		},
+		{
+			"checksumSHA1": "KQBJtydB4I4ko3tFQS9kwBhQiMc=",
+			"path": "github.com/gogo/protobuf/jsonpb",
+			"revision": "5628607bb4c51c3157aacc3a50f0ab707582b805",
+			"revisionTime": "2019-10-14T06:15:17Z"
+		},
+		{
+			"checksumSHA1": "BAVJhQm6l/IS8B1zFSaaihkq+pc=",
 			"path": "github.com/gogo/protobuf/proto",
-			"revision": "43891d9df6604fb88ab1e69e33166f1634646446",
-			"revisionTime": "2020-01-02T12:26:20Z"
+			"revision": "5628607bb4c51c3157aacc3a50f0ab707582b805",
+			"revisionTime": "2019-10-14T06:15:17Z",
+			"version": "v1.3",
+			"versionExact": "v1.3.1"
```

Comment:
We never discussed this before, but we are actually using a different `protobuf/proto` version now than used by `libbeat`. Not quite sure if this could cause some issues.

Reply:
Was that intentional? I can try rolling back to the beats vendored version.

Reply:
afair there were conflicts, as some libraries needed to have a fixed version.